tests/test_psi.py

```diff
@@ -20,5 +20,21 @@
         (([], []), []),
     ],
 )
-def test_compute_psi_returns_correct_indices(test_input, expected):
-    assert expected == compute_psi(*test_input)
+def test_compute_intersection_returns_correct_indices(test_input, expected):
+    client_items = test_input[0]
+    server_items = test_input[1]
+
+    # Handle special case of empty client or server items
+    if len(client_items) == 0 or len(server_items) == 0:
```

Comment:
can we move the error-raising cases to a separate test, to make it clear that we expect different outcomes